### PR TITLE
Implementing support for custom sample apps, locally and via remote git repository

### LINKED ISSUE #22: Add asset flag to use non-built-in sample apps
Add a flag called `--asset` to use a sample app that is not baked into `gonut`, but can be a URL (`file://`, `http://`, or Git repo location).

Example:
`gonut push --asset https://github.com/IBM-Cloud/get-started-node`

Alternatively, test whether the `--asset` flag can be omitted in case the command call itself is unique enough to be understood.
`gonut push https://github.com/IBM-Cloud/get-started-node`

### PR DESCRIPTION
**Pull request by m-lukas**
- Replace sub-command structure (php, golang, all ...) with args
-> add support for specifying multiple apps at one
- Implement url lookup, use of git functionality, local download
- Refactor ExitGonut
- Fix hostname bug in sample app pinging
- Add test cases for localization functions
- Add `all` option to `-s` flag

Fix #22 and #24

### CODE REVIEW

File: internal/gonut/cmd/push.go [hunk 1]
```diff
@@ -123,6 +126,7 @@ var sampleApps = []sampleApp{
 		caption:       ".NET",
 		command:       "dotnet",
 		buildpack:     "dotnet-core",
+		aliases:       []string{"node"},
```
Comment (HeavyWombat) [inline internal/gonut/cmd/push.go]: Is that a copy/paste error? That should not be an alias for that language.

File: internal/gonut/cmd/push.go [hunk 2]
```diff
@@ -131,6 +135,7 @@ var sampleApps = []sampleApp{
 		caption:       "Binary",
 		command:       "binary",
 		buildpack:     "binary_buildpack",
+		aliases:       []string{"node"},
```
Comment (HeavyWombat) [inline internal/gonut/cmd/push.go]: Is that a copy/paste error? That should not be an alias for that language.

File: internal/gonut/cmd/push.go [hunk 3]
```diff
@@ -139,16 +144,21 @@ var sampleApps = []sampleApp{
 		caption:       "Java",
 		command:       "java",
 		buildpack:     "java_buildpack",
+		aliases:       []string{"node"},
```
Comment (HeavyWombat) [inline internal/gonut/cmd/push.go]: Is that a copy/paste error? That should not be an alias for that language.

**Review by HeavyWombat — CHANGES_REQUESTED** [review]
Thanks for the PR. Could you have a look at the additional aliases that seem wrong?

**Comment by m-lukas** [comment]
Fixed :)

**Review by HeavyWombat — APPROVED** [review]

**Pull request merged**